app/assets/stylesheets/scaffolds.scss

```diff
@@ -10,7 +10,7 @@ body {
   padding: 0;
   margin: 0;
   overflow-y: scroll;
-  padding-top: 46px; // height of header
+  padding-top: $header-height; // height of header
```

Comment:
Maybe you don't need the comment anymore since the magic number have been replace with a correctly named variable?

Reply:
Yeap, good catch!